src/wp-includes/html-api/class-wp-html-processor.php

```diff
@@ -2208,7 +2207,7 @@ private function step_in_table(): bool {
 			 * > A DOCTYPE token
 			 */
 			case 'html':
-				// @todo Indicate a parse error once it's possible.
+				// Parse error: ignore the token.
```

Comment:
Your critique of these changes is probably sound. I'm trying to harmonize existing comments and `@todo` statements. I _think_ that in cases of unexpected tokens like this I'm leaving it as `Parse error: ignore the token` mostly to give intention behind why we're returning `$this->step()`. In cases where we have another violation, such as an element not being in scope, I'm leaving the `@todo`. Perhaps we remove them all during 6.7 🤷‍♂️